arangod/Aql/Function.h

```diff
@@ -76,14 +76,17 @@ struct Function {
     /// @brief whether or not the function may read documents from the database
     CanReadDocuments = 16,
 
+    /// @brief whether or not the function is usable in an analyzer
+    CanUseInAnalyzer = 32,
```

Comment:
I hope we don't persist this enum value

Reply:
No, these enum bits are only available at runtime and never persisted.